addons/web/static/src/views/view_button/view_button_hook.js

```diff
@@ -73,15 +86,14 @@ export function useViewButtons(model, ref, options = {}) {
                     Object.assign(buttonContext, clickParams.buttonContext);
                 }
                 const doActionParams = Object.assign({}, clickParams, {
-                    resModel: params.resModel || model.resModel,
-                    resId,
-                    resIds,
-                    context: params.context || {}, //LPE FIXME new Context(payload.env.context).eval();
+                    resModel: resParams.resModel,
+                    resId: resParams.resId,
+                    resIds: resParams.resIds || resParams.resId || [resParams.resId],
+                    context: resParams.context || {}, //LPE FIXME new Context(payload.env.context).eval();
```

Comment:
LPE's fixme is there for so long, the code is probably fine, so I think we can remove it